CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next
+
+- **Breaking**: Remove deprecated `autoregister` and `autoenqueue` methods.
```

Comment:
@kadamwhite Does `register_assets` removal need to be mentioned as well, considering it's being removed in this commit?

Reply:
Great catch, @fsylum, I've made that change!